src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -290,6 +267,12 @@ function ComposerWithSuggestions(
     });
     const commentRef = useRef(value);
 
+    const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`);
+    const [modal] = useOnyx(ONYXKEYS.MODAL);
+    const [preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE] = useOnyx(ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE, {selector: EmojiUtils.getPreferredSkinToneIndex});
+    const [editFocused] = useOnyx(ONYXKEYS.INPUT_FOCUSED);
+    const [parentReportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${parentReportID}`, {canEvict: false, initWithStoredValues: false});
```

Comment:
@blazejkustra I think we have a blocker issue here if `parentReportID` is undefined in which case the app will crash. We can reproduce this if we search for a chat with a user for whom there is no report present locally. Here is a screenshot of the crash. 

![Screenshot 2024-11-04 at 10 26 55 PM](https://github.com/user-attachments/assets/b701ae0d-39bb-45d1-9f89-df017c9c5ec2)

@lakchote Not sure if we can have a fix soon for this. Otherwise we will have to revert the PR as this will be a blocker. Sorry for the inconvenience caused.